### PR TITLE
fixed issue #18

### PR DESCRIPTION
**Pull request by inyourface34456**
changed the controller to an option

### CODE REVIEW

File: plastic_tui/src/ui.rs [hunk 1]
```diff
@@ -451,14 +451,19 @@ impl Ui {
 
     fn handle_gamepad(&mut self) {
         // set events in the cache and check if gamepad is still active
-        while let Some(GilrsEvent { id, event, .. }) = self.gilrs.next_event() {
+        if self.gilrs.is_none() {
```
Comment (Amjad50) [inline plastic_tui/src/ui.rs:454]: A small improvement here,
you can use 
```rust
// didn't test the code, but the idea here
let Some(mut gilrs) = self.gilrs else {
    return;
}
```
then afterwards, you can use `girls` without the need for `as_mut().unwrap()`

**Comment by Amjad50** [comment]
Also please run `cargo fmt` before the next commit, looks like its failing the CI.
Also check `cargo clippy`

File: plastic_tui/src/ui.rs (hunk 1 above)
Comment (inyourface34456) [reply plastic_tui/src/ui.rs:454]: Thanks for the suggestion, will implement

**Comment by inyourface34456** [comment]
> Also please run `cargo fmt` before the next commit, looks like its failing the CI. Also check `cargo clippy`

Fixed all of the issues.

File: plastic_tui/src/ui.rs (hunk 1 above)
Comment (Amjad50) [reply plastic_tui/src/ui.rs:454]: You can use ref

**Pull request merged**